src/main/java/uk/gov/hmcts/reform/sscs/ccd/domain/Adjournment.java

```diff
@@ -0,0 +1,95 @@
+    private LocalDate directionsDueDate;
+    @JsonProperty("adjournCaseTypeOfNextHearing")
+    private AdjournCaseTypeOfHearing typeOfNextHearing;
+    @JsonProperty("adjournCaseNextHearingVenue")
+    private AdjournCaseNextHearingVenue nextHearingVenue;
+    @JsonProperty("adjournCaseNextHearingVenueSelected")
+    private DynamicList nextHearingVenueSelected;
+    @JsonProperty("adjournCasePanelMembersExcluded")
+    private AdjournCasePanelMembersExcluded panelMembersExcluded;
+    @JsonProperty("adjournCaseDisabilityQualifiedPanelMemberName")
+    private String disabilityQualifiedPanelMemberName;
+    @JsonProperty("adjournCaseMedicallyQualifiedPanelMemberName")
+    private String medicallyQualifiedPanelMemberName;
+    @JsonProperty("adjournCaseOtherPanelMemberName")
+    private String otherPanelMemberName;
+    @JsonProperty("adjournCaseNextHearingListingDurationType")
+    private AdjournCaseNextHearingDurationType nextHearingListingDurationType;
+    @JsonProperty("adjournCaseNextHearingListingDuration")
+    private Integer nextHearingListingDuration;
+    @JsonProperty("adjournCaseNextHearingListingDurationUnits")
+    private AdjournCaseNextHearingDurationUnits nextHearingListingDurationUnits;
+    @JsonProperty("adjournCaseInterpreterRequired")
+    private YesNo interpreterRequired;
+    @JsonProperty("adjournCaseInterpreterLanguageList")
+    private String interpreterLanguage;
+    @JsonProperty("adjournCaseNextHearingDateType")
+    private AdjournCaseNextHearingDateType nextHearingDateType;
+    @JsonProperty("adjournCaseNextHearingDateOrPeriod")
+    private AdjournCaseNextHearingDateOrPeriod nextHearingDateOrPeriod;
+    @JsonProperty("adjournCaseNextHearingDateOrTime")
+    private String nextHearingDateOrTime;
+
+    @JsonFormat(shape = JsonFormat.Shape.STRING, pattern = "yyyy-MM-dd")
+    @JsonSerialize(using = LocalDateSerializer.class)
+    @JsonProperty("adjournCaseNextHearingFirstAvailableDateAfterDate")
+    private LocalDate nextHearingFirstAvailableDateAfterDate;
+    @JsonProperty("adjournCaseNextHearingFirstAvailableDateAfterPeriod")
+    private AdjournCaseNextHearingPeriod nextHearingFirstAvailableDateAfterPeriod;
+    @JsonProperty("adjournCaseTime")
+    private AdjournCaseTime time;
+    @JsonProperty("adjournCaseReasons")
+    private List<CollectionItem<String>> reasons;
+    @JsonProperty("adjournCaseAdditionalDirections")
+    private List<CollectionItem<String>> additionalDirections;
+
+    @DocumentLinkMustBePdf(message = "You need to upload PDF documents only")
+    @JsonProperty("adjournCasePreviewDocument")
+    private DocumentLink previewDocument;
+
+    @JsonFormat(shape = JsonFormat.Shape.STRING, pattern = "yyyy-MM-dd")
+    @JsonSerialize(using = LocalDateSerializer.class)
+    @JsonProperty("adjournCaseGeneratedDate")
+    private LocalDate generatedDate;
+    @JsonProperty("isAdjournmentInProgress")
+    private YesNo isAdjournmentInProgress;
```

Comment:
It could be renamed to adjournmentInProgress and therefore getAdjournmentInProgress. Postponement has unprocessedPostponement which serves the same purpose.